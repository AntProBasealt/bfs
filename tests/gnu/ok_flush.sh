# I/O streams should be flushed before -ok prompts
yes | invoke_bfs basic -printf '%p ? ' -ok echo found \; 2>&1 | tr '\0' ' ' | sed 's/?.*?/?/' >"$OUT"
sort_output
diff_output
