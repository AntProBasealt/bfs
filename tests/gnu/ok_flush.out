basic ? found
basic/a ? found
basic/b ? found
basic/c ? found
basic/c/d ? found
basic/e ? found
basic/e/f ? found
basic/g ? found
basic/g/h ? found
basic/i ? found
basic/j ? found
basic/j/foo ? found
basic/k ? found
basic/k/foo ? found
basic/k/foo/bar ? found
basic/l ? found
basic/l/foo ? found
basic/l/foo/bar ? found
basic/l/foo/bar/baz ? found
